app/models/privacy_policy.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+class PrivacyPolicy < ApplicationRecord
+  class Acceptance < ApplicationRecord
+    self.table_name = :privacy_policy_acceptances
+
+    belongs_to :user
+    belongs_to :privacy_policy
+  end
+
+  def self.current
+    order(:major_version, :minor_version).last
+  end
+
+  def acceptance_required?(user)
+    return false if !user || user.admin?
+    return false unless user.induction_coordinator_profile
```

Comment:
Does this simplify to `return false unless user&.induction_coordinator?`?

Reply:
Theoretically, but induction coordinator can in theory be an admin (at least db structure does not prohibit that). Happy to update if we are certain this will never be the case